frontend/src/components/GcsLink.tsx

```diff
@@ -6,11 +6,22 @@ import { generateGcsConsoleUri } from '../lib/Utils';
  * text if it is not a valid gs:// uri.
  */
 export const GcsLink: React.FC<{ gcsUri?: string }> = ({ gcsUri }) => {
-  const gcsConsoleUri = gcsUri ? generateGcsConsoleUri(gcsUri) : undefined;
-  if (gcsConsoleUri) {
+  var clickableUrl: string | undefined = undefined
```

Comment:
In modern typescript, best practice is to never use var. Use let if mutable, use const if immutable.

Reply:
Fixed.